curriculum/challenges/english/07-scientific-computing-with-python/learn-special-methods-by-building-a-vector-space/65fd681b20b7e45f55def415.md

```diff
@@ -0,0 +1,89 @@
+---
+id: 65fd681b20b7e45f55def415
+title: Step 55
+challengeType: 20
+dashedName: step-55
+---
+
+# --description--
+
+The scalar multiplication works fine. Now, modify the assignment of `v5` to be the dot product `v1 * v2`. Also, update the `print` call.
```

Comment:
Try to multiply a R2Vector and a R3Vector.
```py
f = R2Vector(x=1, y=2)
g = R3Vector(x=3, y=8, z=1)

f * g
g * f
```
The two multiplications should both raise the error, only one of the two does it.

This may happen everytime `isinstance` is used.